gateway/api.go

```diff
@@ -847,6 +847,236 @@ func apiHandler(w http.ResponseWriter, r *http.Request) {
 	doJSONWrite(w, code, obj)
 }
 
+func policiesHandler(w http.ResponseWriter, r *http.Request) {
+	policyID := mux.Vars(r)["policyID"]
+
+	var obj interface{}
+	var code int
+
+	switch r.Method {
+	case "GET":
```

Comment:
Better to use `http.MethodGet` and similar constants.

Reply:
Sure